fec.config.js

```diff
@@ -139,6 +139,10 @@ module.exports = {
         __dirname,
         '/src/modules/IOPInventoryTable.js',
       ),
+      './SystemsTable': resolve(
+        __dirname,
+        'src/routes/Systems/components/SystemsTable/SystemsTable.js',
```

Comment:
Should we have a wrapper in `modules`? I kinda like the pattern as it's easy to see what components are exposed, but not a deal breaker.